src/runtime/math.py

```diff
@@ -139,3 +139,27 @@ def lcm(a: i32, b: i32) -> i32:
     if a*b == 0:
         return 0
     return (a*b)//gcd(a, b)
+
+def sqrt(x: f64) -> f64:
+    return x**(1/2)
+
+@ccall
+def _lfortran_dsin(x: f64) -> f64:
+    pass
+
+def sin(x: f64) -> f64:
+    return _lfortran_dsin(x)
+
+@ccall
+def _lfortran_dcos(x: f64) -> f64:
+    pass
+
+def cos(x: f64) -> f64:
+    return _lfortran_dcos(x)
```

Comment:
Why are we mixing `pure` and `impure` implementations?

Reply:
Right now everything is in one directory. This would be impure, because it depends on C.

Reply:
OK, we can separate them down the road.